lua/nvim-lsp-installer/servers/nimls/init.lua

```diff
@@ -0,0 +1,21 @@
+local server = require "nvim-lsp-installer.server"
+local shell = require "nvim-lsp-installer.installers.shell"
+
+return function(name, root_dir)
+  return server.Server:new {
+    name = name,
+    root_dir = root_dir,
+    homepage = "https://github.com/PMunch/nimlsp",
+    languages = { "nim" },
+    installer = {
+      shell.bash [[ 
+        git clone --depth 1 https://github.com/PMunch/nimlsp.git .
+        nimble build
+      ]]
+    },
```

Comment:
To make this OS-agnostic, we should avoid using the `shell` installers altogether (I don't believe they're used anywhere in this plugin anymore, they'll most likely be removed soon).

Something like this ought to do it (I added the `context.receipt()` call, which will make sure that write a receipt with installation information to a JSON file upon completion.

```suggestion
    installer = {
      std.git_clone "https://github.com/PMunch/nimlsp.git",
      function (_, callback, ctx)
        process.spawn("nimble", {
          args = { "build" },
          cwd = ctx.install_dir,
          stdio_sink = ctx.stdio_sink,
        }, callback)
      end,
      context.receipt(function (receipt)
        receipt:with_primary_source(receipt.git_remote "https://github.com/PMunch/nimlsp.git")
      end)
    }
```